src/pyhf/pdf.py

```diff
@@ -650,7 +650,21 @@ def logpdf(self, maindata, pars):
         """
         return self.make_pdf(pars).log_prob(maindata)
 
-    def _modifications(self, pars):
+    def modifications(self, pars):
+        """
+        Obtain the additive and multiplicative modifications to the expected
+        event rates for the model parameters.
+
+        Args:
+            pars (:obj:`tensor`): The model parameters
+
+        Returns:
+            Tuple of additive and multiplicative modifications:
+                - deltas (:obj:`list`) is the result of an `apply(pars)` of combined modifiers
+                  with 'addition' op_code
+                - factors (:obj:`list`) is the result of `apply(pars)` of combined modifiers
+                  with 'multiplication' op_code
```

Comment:
I realize that you were just taking the style from the rest of this file, but this file is old and doesn't really work with our current rendering, so to make things appear code-like we can just throw on some additional backticks:

```suggestion
                - deltas (:obj:`list`) is the result of an ``apply(pars)`` of combined modifiers
                  with ``"addition"`` ``op_code``
                - factors (:obj:`list`) is the result of ``apply(pars)`` of combined modifiers
                  with ``"multiplication"`` ``op_code`
```

Reply:
Render of the suggested changes:

![image](https://github.com/scikit-hep/pyhf/assets/5142394/cce1c079-56eb-48f3-bfd5-78ef19022281)